packages/ubuntu_desktop_installer/lib/routes.dart

```diff
@@ -1,5 +1,5 @@
 abstract class Routes {
-  static const initial = welcome;
+  static const initial = installationType;
```

Comment:
Did you intend to change the initial route in this PR as well?

Reply:
Hah, and not the first time because it's so convenient for testing. :D I'm glad there's a test that catches this.